controllers/new.go

```diff
@@ -594,8 +594,10 @@ func (k *kubeResources) newPolicyEngineContainer(instance *cv1.UpdateService, im
 			PeriodSeconds:       10,
 			TimeoutSeconds:      3,
 			Handler: corev1.Handler{
-				TCPSocket: &corev1.TCPSocketAction{
-					Port: intstr.FromInt(8081),
+				HTTPGet: &corev1.HTTPGetAction{
+					Path:   "/metrics",
+					Port:   intstr.FromInt(9081),
```

Comment:
[e2e][1]:

```
I0105 17:21:47.971812    5751 utils.go:106] Waiting for full availability of example deployment (0/1)
    updateservice_creation_test.go:81: timed out waiting for the condition
```

Because these probes are 503ing:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cincinnati-operator/139/pull-ci-openshift-cincinnati-operator-master-operator-e2e/1478769955875852288/artifacts/operator-e2e/gather-extra/artifacts/nodes/ci-op-in4lfpdq-fb094-vx4mk-worker-c-wwvk4/journal | zgrep 'example.*probe' | tail -n1
Jan 05 17:21:48.150903 ci-op-in4lfpdq-fb094-vx4mk-worker-c-wwvk4 hyperkube[1425]: I0105 17:21:48.150428    1425 prober.go:116] "Probe failed" probeType="Readiness" pod="openshift-updateservice/example-56cc96c9fc-ktflx" podUID=7ccae62d-aec1-4c95-99bf-1d078ac1e248 containerName="graph-builder" probeResult=failure output="HTTP probe failed with statuscode: 503"
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-operator/139/pull-ci-openshift-cincinnati-operator-master-operator-e2e/1478769955875852288#1:build-log.txt%3A80

Reply:
>containerName="graph-builder"

That's GB probe, not PE

Reply:
GB initial scrape gets stuck at
>[2022-01-06T10:12:45Z ERROR graph_builder::graph] Concrete release with the same version (4.7.32+amd64) already exists

Reply:
Checking if updated Cincinnati would resolve that - https://github.com/openshift/release/pull/25027